packages/wxt/src/core/resolve-config.ts

```diff
@@ -289,7 +289,7 @@ function resolveZipConfig(
   return {
     name: undefined,
     sourcesTemplate: '{{name}}-{{version}}-sources.zip',
-    artifactTemplate: '{{name}}-{{version}}-{{browser}}.zip',
+    artifactTemplate: mergedConfig.artifactTemplate ?? '{{name}}-{{version}}-{{browser}}.zip',
```

Comment:
Hmm, we spread the `mergedConfig` a few lines down. That should overwrite the `artifactTemplate` option. Does that not work for you?

Reply:
Aaahh, I'm blind, appreciating the hint, works perfectly fine ♥️